test/unit/alignment/matrix/detail/two_dimensional_matrix_test.cpp

```diff
@@ -30,30 +30,26 @@ using test_matrix_t = seqan3::detail::two_dimensional_matrix<score_t, allocator_
 template <typename score_type, matrix_major_order order = matrix_major_order::row>
 std::vector<score_type, std::allocator<score_type>> create_matrix_storage()
 {
-    // this is a small hack to allow simd types in an initialiser list on gcc 7;
-#if defined(__GNUC__) && (__GNUC__ == 7)
+    // this is a small hack to allow simd types in an initialiser list on gcc 7 and gcc 10;
     using storage_t = std::array<score_type, 12>;
-#else
-    using storage_t = std::vector<score_type>;
-#endif
```

Comment:
Mhm, so what is the problem here and which version should work? So what case was the workaround?

Reply:
* gcc7 `std::vector<score_type>;` did not work
* gcc8 `std::array<score_type, 12>` did not work without extra `{}` when using initialiser lists
* gcc9 same as gcc8
* gcc10 same as gcc7

I found now a syntax that works on all versions, but uses `std::array<score_type, 12>`